app/gateway/2.6.x/get-started/quickstart/adding-consumers.md

```diff
@@ -8,7 +8,8 @@ associated to individuals using your Service, and can be used for tracking, acce
 management, and more.
 
 ## Before you start
-* You have installed and started {{site.base_gateway}}, either through the [Docker quickstart](/gateway/{{page.kong_version}}/get-started/quickstart) or a more [comprehensive installation](/gateway/{{page.kong_version}}/install-and-run).
+
+* You have installed and started {{site.base_gateway}}, either through the [Docker quickstart](/gateway/{{page.kong_version}}/get-started/quickstart) or a more [comprehensive installation](/gateway/{{page.kong_version}}/get-started/comprehensive/).
```

Comment:
```suggestion
* You have installed and started {{site.base_gateway}}, either through the [Docker quickstart](/gateway/{{page.kong_version}}/get-started/quickstart) or a more [comprehensive installation](/gateway/{{page.kong_version}}/install-and-run).
```
Setting this back to `install-and-run`; the guide you're changing it to doesn't include install instructions.